python/src/spark_rapids_ml/clustering.py

```diff
@@ -959,6 +960,13 @@ def _cuml_fit(
             dbscan.n_cols = params[param_alias.num_cols]
             dbscan.dtype = np.dtype(dtype)
 
+            # get_logger(self.__class__).info("Row", concated.shape)
```

Comment:
best to delete this commented code.

Reply:
Oops, forgot to delete the code used to detect the out of memory stuff. Sorry about that.